src/message.rs

```diff
@@ -951,13 +981,15 @@ pub fn get_mime_headers(context: &Context, msg_id: MsgId) -> Option<String> {
 }
 
 pub fn delete_msgs(context: &Context, msg_ids: &[MsgId]) {
-    for msg_id in msg_ids.iter() {
+    for msg_id in msg_ids {
         if let Ok(msg) = Message::load_from_db(context, *msg_id) {
             if msg.location_id > 0 {
                 delete_poi_location(context, msg.location_id);
             }
         }
-        update_msg_chat_id(context, *msg_id, ChatId::new(DC_CHAT_ID_TRASH));
+        if let Err(err) = msg_id.trash(context) {
+            warn!(context, "Unable to trash message {}: {}", msg_id, err);
```

Comment:
Should the user be aware of this?  Could it mean a message they thought would no longer exist on local storage still does.
I.e., should this be `error!` instead of `warn!`?

Reply:
Changed to `error!`